internal/commands/push/command.go

```diff
@@ -141,11 +141,6 @@ func (cmd *Command) Inputs() cli.InputResolver {
 
 // Handler is the command handler
 func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.Clients) error {
-	app, err := local.LoadApp(cmd.inputs.LocalPath)
```

Comment:
⚠️

not sure I love this, but removed this as an optimization b/c we also called `local.FindApp` in the `inputs.Resolve` phase.  nothing really changes with the command logic itself, but a side effect of this made testing fairly nasty (see: the changes in `push/command_test.go`)

we can add this back or live with the weird `Handler` testing

Reply:
😬 I am also not fond of having to either `LoadApp` or `Resolve` before setting the `cmd` in the tests

Reply:
convinced me... reverted that optimization